net/speedtest/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+version             1.1.1.84
+revision            0
+
+name                speedtest
+categories          net
+platforms           darwin
+supported_archs     noarch
+maintainers         {@linuxgemini linuxgemini.space:macportspackaging} openmaintainer
+
+homepage            https://speedtest.net/apps/cli
+license             restrictive nomirror noncommercial
+
+description         official cli for speedtest.net
+long_description    command line interface for testing internet bandwidth using speedtest.net
+
+master_sites        https://install.speedtest.net/app/cli/
+distname            ookla-speedtest-${version}-macosx-x86_64
+extract.suffix      .tgz
+
+# ookla already ships this as an universal binary
+universal_variant   no
```

Comment:
Since the binary is universal, the port should have a universal variant. Either the port can force the universal variant on, to convey that the software that is installed is universal, or ideally the universal variant can be optional, and if the user does not select it, then the port uses `lipo` to remove the architectures that were not requested.

Reply:
TIL about `lipo`, thank you.

Resolved in b64b2932c58a1adfdb20129e1234c07b65dcb6cc.